package.json

```diff
@@ -1,100 +1,94 @@
 {
   "name": "@payfit/multi-semantic-release",
-  "author": "Dave Houlbrooke <dave@shax.com>",
-  "version": "3.16.0-alpha.22",
+  "author": "PayFit",
+  "version": "0.0.0-development",
   "license": "0BSD",
   "engines": {
-    "node": ">=10.18",
+    "node": ">=12",
     "yarn": ">=1.0.0"
   },
-  "main": "dist/src/lib/multiSemanticRelease.js",
+  "main": "dist/lib/multiSemanticRelease.js",
```

Comment:
How do we resolve the version now ? It means that you won't have the package.json in your build anymore, right ?

Reply:
Exactly. Was painful just to log the version I think

Reply:
Did you adapted the code somewhere ? I probably missed it between all the linting changes 🙈

Reply:
there was nothing to adapt, just remove the line in the cli file where we looked at the version.
The rest was done in the tsconfig.json file